webviz_subsurface/containers/_summary.py

```diff
@@ -0,0 +1,670 @@
+        color: str = trace color
+    Returns:
+        plotly-graph-obj-tracegroup
+    """
+
+    ens_traces = []
+
+    # 1st and only trace of the legendgroup to show up in legend
+    ens_traces.append({
+        'x': ens_smry_data[ens_smry_data['REAL']
+                           == ens_smry_data.REAL.unique()[0]]['DATE'],
+        'y': ens_smry_data[ens_smry_data['REAL']
+                           == ens_smry_data.REAL.unique()[0]][vector],
+        'legendgroup': ens,
+        'name': ens,
+        'type': 'markers',
+        'marker': {
+            'color': color
+        },
+        'showlegend': True
+    })
+
+    for real in ens_smry_data.REAL.unique()[1:]:
+
+        ens_traces.append({
+            'x': ens_smry_data[ens_smry_data['REAL'] == real]['DATE'],
+            'y': ens_smry_data[ens_smry_data['REAL'] == real][vector],
+            'legendgroup': ens,
+            'name': ens,
+            'type': 'line',
+            'marker': {
+                'color': color
+            },
+            'showlegend': False
+        })
+
+    return ens_traces
+
+
+def single_trace(
+        ens_smry_data,
+        ens: str,
+        vector: str,
+        color: str):
+    """ function to create a single trace that shows up in the legend.
+
+    Args:
+        smry_data: pd.df = calculated summary data (fmu-ensemble)
+        ens: str = selected ensemble
+        color str = color
+    Returns:
+        trace-obj to be plotted in a plotly.fig
+    """
+
+    return {
+        'x': ens_smry_data[ens_smry_data['REAL']
+                           == ens_smry_data.REAL.unique()[0]]['DATE'],
+        'y': ens_smry_data[ens_smry_data['REAL']
+                           == ens_smry_data.REAL.unique()[0]][vector],
+        'legendgroup': ens,
+        'name': ens,
+        'type': 'markers',
+        'marker': {
+            'color': color
+        },
+        'showlegend': True
+    }
+
+
+def time_series_confidence_interval_traces(
+        vector_stats,
+        color_rgb: list,
+        legend_group: str
+        ):
+    """ function to create a convidence interval set of a selected ensemble.
+
+    Args:
+        smry_data: pd.df = calculated summary data (fmu-ensemble)
+        color (r: int, g:int, b:int) = rgb of color to be ploted
+        legend_group: str = selected ensemble
+    Returns:
+        list if trace-obj representing min, max, p10, p90 and mean
+    """
+
+    r, g, b = color_rgb
+
+    trace_maximum = go.Scatter(
+        name='maximum',
+        x=vector_stats['maximum'].index.tolist(),
+        y=vector_stats['maximum'].values,
+        mode='lines',
+        line=dict(width=0),
+        legendgroup=legend_group,
+        showlegend=False,
+    )
+
+    trace_p10 = go.Scatter(
+        name='p10',
+        x=vector_stats['p10'].index.tolist(),
+        y=vector_stats['p10'].values,
+        mode='lines',
+        fill='tonexty',
+        fillcolor='rgba({},{},{},{})'.format(r, g, b, 0.3),
```

Comment:
Replace these types of strings with `f'rgba({r}, {g}, {b}, 0.3)'`

Reply:
Changed